docs/build_version_doc/build_doc.sh

```diff
@@ -49,12 +49,12 @@ then
 fi
 
 # Build new released tag
-if [ $latest_tag != ${tag_list[0]} ]
+if [ $latest_tag == ${tag_list[0]} ] # this is a temporary fix changed from != to == (SHOULD BE REMOVED LATER)
```

Comment:
Some other things that happen later, you don't want to trigger. This will create a v1.1.0 folder down the line, and we don't want that.

These lines probably should be commented out when you run it for v1.1.0:
https://github.com/thinksanky/incubator-mxnet/blob/a982b14c4424852ad524cb89440dd23252803c8e/docs/build_version_doc/build_doc.sh#L66-L69
Maybe keep the README line...